renovate/src/Renovate/Redirect/Symbolize.hs

```diff
@@ -81,8 +82,11 @@ symbolizeJumps isa mem symAddrMap (cb, symAddr) =
   case DLN.nonEmpty (concat insns) of
     Nothing -> error ("Created empty block while symbolizing block at: " ++ show (concreteBlockAddress cb))
     Just insnList ->
-      (cb, symbolicBlock (concreteBlockAddress cb) symAddr insnList)
+      let msucc = lookupSymAddr =<< explicitFallthroughSuccessor fallthrough
+      in (cb, symbolicBlock cb symAddr insnList msucc)
```

Comment:
Hm, is this really the right behavior? The way I read this, it says that if there's an explicit fallthrough address in `fallthrough`, but we can't find that address in our `symAddrMap`, we claim that no fallthrough is needed. Maybe we should report an error of some kind instead?

Reply:
Good catch - you are right